koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -520,7 +528,8 @@ def runBuilds(self, src, target_info, arches, scratch=False,
                                                                 scratch,
                                                                 yum_repourls,
                                                                 branch,
-                                                                push_url],
+                                                                push_url,
+                                                                labels],
```

Comment:
Is the default `None` value valid here? If not, we should use something like `labels or {}`.